examples/xml/200shellComputationAnevrism.scn

```diff
@@ -27,22 +55,22 @@
     <Node name="Shell">
         <EulerImplicitSolver />
         <!--<CGLinearSolver iterations="20" tolerance="0.05" threshold="1e-20"/>-->
-        
-        <PCGLinearSolver iterations="5" tolerance="0.05"  preconditioners="Precond"/>
-        <SparseLDLSolver name="Precond"/>
+        <LCPConstraintSolver/>
```

Comment:
You can move the LCPConstraintSolver just below the `FreeMotionAnimationLoop`. They work together